pkg/db/models/metrics.go

```diff
@@ -0,0 +1,6 @@
+package models
+
+type MetricsRefresh struct {
```

Comment:
How about a more generic LastUpdate, as there could be other uses for parts of sippy that want to track when something was last refreshed/updated.